### PR TITLE
CY-3470 - Update Cloudify Composer documentation

### PR DESCRIPTION
**Pull request by qooban**
* Reorganized Cloudify Composer section
  * Before
    image
  * After
    image
* Started using parameters for Cloudify Console, Cloudify Composer and Cloudify Manager
* Updated screenshots according to current state
* Fixed many small issues

### CODE REVIEW

File: content/developer/composer/blueprint-creation.md [hunk 1]
```diff
@@ -1,122 +1,180 @@
...
+### Setting interfaces
+
+To select the operations that you require, click the ![Select Operation]( /images/composer/select-implementation-icon.png ) icon next to the implementation fields in the node's Interfaces section on the right of the screen. The following dialog box is displayed:
+
+![Implementation tree]( /images/composer/implementation-tree.png )
+
+
+### Defining relationships
+
+To define a relationships between nodes, where the connector icon is displayed, draw a connecting line from the edge of one node type to the edge of another. Note that the connector icons show either relationships in or relationships out of a node type.
```
Comment (kubama) [inline content/developer/composer/blueprint-creation.md:78]:
```suggestion
To define a relationship between nodes, where the connector icon is displayed, draw a connecting line from the edge of one node type to the edge of another. Note that the connector icons show either relationships in or relationships out of a node type.
```

File: content/developer/composer/blueprint-creation.md [hunk 2]
```diff
@@ -1,122 +1,180 @@
...
+You can group a number of components using the ![drag to select]( /images/composer/drag-to-select.png ) button. Select the required nodes and click on the ![create node group]( /images/composer/create-node-group.png ) button to create a resource group in the Topology view. 
+
+The resource group is also created in the source code. You can click the group to display its properties and add or remove members.
+
+![create group]( /images/composer/create-group.png )
+
+
+### Intrinsic Functions
+
+As in {{< param mgr_premium_title >}}, the values of a node’s properties, inputs or outputs can be specified as intrinsic function return values. The intrinsic functions list is available at [Intrinsic functions specification page]({{< relref "developer/blueprints/spec-intrinsic-functions.md" >}}).
```
Comment (kubama) [inline content/developer/composer/blueprint-creation.md:133]:
```suggestion
As in {{< param mgr_premium_title >}}, the values of node properties, inputs or outputs can be specified as intrinsic function return values. The intrinsic functions list is available at [Intrinsic functions specification page]({{< relref "developer/blueprints/spec-intrinsic-functions.md" >}}).
```

File: content/developer/composer/features.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+## Importing nodes
+
+Before you start to design your blueprint, you will typically want to add the stencils that contain the basic node types that you need to work with, and the operations that they expose. For more information about node types, [click here]({{< relref "developer/blueprints/spec-node-types.md" >}}).
+
+When you add a plugin as an import to {{< param cfy_composer_name >}}, both the nodes types and the operations that the plugin exposes are supported in the blueprint. See [Managing Plugins]({{< relref "developer/composer/managing-plugins.md" >}}) on how to add plugins to a blueprint.
+
+
+## Working with nodes
+
+You add a node by dragging the required node type from the Stencils pane and dropping it on to the canvas. You then click it to edit its properties. The properties that are available are dependent on the node type.
```
Comment (kubama) [inline content/developer/composer/features.md:21]:
```suggestion
You add a node by dragging the required node type from the Stencils pane and dropping it onto the canvas. You then click it to edit its properties. Available properties depend on the node type.
```

File: content/developer/composer/features.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+Every addition or change that you make to the topology of your Blueprint package is reflected in code of the main blueprint YAML file. 
+
+See [Source view]({{< relref "developer/composer/blueprint-creation.md#source-view" >}}) for more details on working with source code.
+
+
+## Handling resources
+
+You can organize your blueprint package structure according to your needs. You can upload files and group them into folders, you can edit files inside {{< param cfy_composer_name >}}.
+
+See [Managing Resources]({{< relref "developer/composer/managing-resources.md" >}}) on how to add/remove files/folders to a blueprint.
```
Comment (kubama) [inline content/developer/composer/features.md:41]:
```suggestion
See [Managing Resources]({{< relref "developer/composer/managing-resources.md" >}}) on how to add/remove files/folders from a blueprint.
```

File: content/developer/composer/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+Starting from Cloudify 4.1, the {{< param cfy_composer_name >}} comes as part of the {{< param mgr_premium_title >}}.
+
+
+## Starting
+
+You can access {{< param cfy_composer_name >}} from your browser:
+ 
+ * directly via `http://<cloudify-manager-ip>/composer` or `https://<cloudify-manager-ip>/composer` 
+ * from {{< param cfy_console_name >}} by clicking on the **{{< param cfy_composer_name >}}** link button at Local Blueprints page
+
```
Comment (kubama) [inline content/developer/composer/getting-started.md:20]: Maybe it's worth to mention "Edit a copy in composer" feature here?

File: content/developer/composer/getting-started.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+{{< param cfy_composer_name >}} uses the {{< param mgr_premium_title >}} user definitions and to access the {{< param cfy_composer_name >}}, you have to login to the {{< param cfy_console_name >}}. 
+
+
+## Main page
+
+When you have logged in, the default Topology page is displayed, with an empty blueprint for your use. The left side of the {{< param cfy_composer_name >}} screen displays:
+
+1. Project View pane that contains list of all blueprint packages created within {{< param cfy_composer_name >}}
```
Comment (kubama) [inline content/developer/composer/getting-started.md:28]: created within or imported into

File: content/developer/composer/getting-started.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+
+
+## Main page
+
+When you have logged in, the default Topology page is displayed, with an empty blueprint for your use. The left side of the {{< param cfy_composer_name >}} screen displays:
+
+1. Project View pane that contains list of all blueprint packages created within {{< param cfy_composer_name >}}  
+2. Stencils pane that contains node types that can be used as the building blocks of the topology. 
+
+By default the main pane is a canvas on to which you can drag and drop nodes and define the relationships between them.
```
Comment (kubama) [inline content/developer/composer/getting-started.md:31]:
```suggestion
By default the main pane is a canvas onto which you can drag and drop nodes and define the relationships between them. 
```

File: content/developer/composer/getting-started.md [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+Under each blueprint you have the following navigation options:
+
+* **Topology** - see [Creating blueprint]({{< relref "developer/composer/blueprint-creation.md" >}}) for details
+* **Resources** - see [Managing resources]({{< relref "developer/composer/managing-resources.md" >}}) for details
+* **Inputs & Outputs** - see [Managing Inputs and Outputs]({{< relref "developer/composer/managing-inputs-outputs.md" >}}) for details
+* **Plugins** - see [Managing Plugins]({{< relref "developer/composer/managing-plugins.md" >}}) for details
+
+#### Global actions
+
+At the top of the list are buttons to enable you to create or import a blueprint.
```
Comment (kubama) [inline content/developer/composer/getting-started.md:50]:
```suggestion
At the top of the list there are buttons to enable you to create or import a blueprint. 
```

File: content/developer/composer/getting-started.md [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+Uploading enables you to select to which of the tenants on the {{< param mgr_premium_title >}} you want the blueprint to be uploaded. You can only upload to tenants that your user credentials give you permission to access.
+
+The download operation downloads the last saved blueprint package as a TAR.GZ or ZIP archive.
+
+Validating a blueprint reviews the source code, to ensure that logical concepts are valid.
+
+
+### Stencils
+
+All available node types that your current blueprint can useare displayed in the left pane, called **Stencils**.
```
Comment (kubama) [inline content/developer/composer/getting-started.md:76]:
```suggestion
All available node types that your current blueprint can use are displayed in the left pane, called **Stencils**. 
```

File: content/developer/composer/getting-started.md (hunk 5 above)
Comment (qooban) [reply content/developer/composer/getting-started.md:23]: Right, done.

File: content/developer/composer/getting-started.md (hunk 6 above)
Comment (qooban) [reply content/developer/composer/getting-started.md]: Fixed.

**Review by kubama — APPROVED** [review]

**Review by ofercloudify — APPROVED** [review]

**Pull request merged**